docs: Adding clarifications to the Quickstart and Installation (#3782)
werf/werf: docs/site/_includes/en/installation/trdl_linux.md

Make sure you have Git 2.18.0 or newer and [Docker](https://docs.docker.com/get-docker) installed.

> To run `werf` as a regular user you should have access to the Docker daemon. Create a user group `docker`. Please note that Docker itself creates this group during installation, and the command below is needed for safety reasons.

Reviewer: > Please note that Docker itself creates this group during installation, and the command below is needed for safety reasons.

Неважная инфа, которая ничего не менят для пользователя, имхо лучше не перегружать.

Reviewer: ```suggestion
To run `werf` as a regular user, you should have access to the Docker daemon. First, make sure the `docker` group exists:
```

```shell
sudo groupadd docker
```

> If after execution the message `groupadd: group 'docker' already exists` appears, then such a group has already been created earlier.

Reviewer: ```suggestion
```
Сообщение об ошибке уже довольно лаконично сообщает об этом.

> Add your current user to the `docker` group.

Reviewer: ```suggestion
Add your current user to the `docker` group:
```

trdl add werf https://tuf.werf.io 1 b7ff6bcbe598e072a86d595a3621924c8612c7e6dc6a82e919abe89707d7e3f468e616b5635630680dd1e98fc362ae5051728406700e6274c5ed1ad92bea52a2
```

> Activate `werf` and add automatic activation for new shell sessions:

Reviewer: > For local usage we recommend

Это важно, лучше вернуть. Эти две команды не всегда понадобятся. Либо эти две команды, либо следующий блок (_**But** in CI you should prefer..._), одно из двух.
